src/cr/cube/matrix/measure.py

```diff
@@ -945,23 +1053,25 @@ def blocks(self):
     @lazyproperty
```

Comment:
@malecki - change to mean significance formula here.

The big change is that I'm also using a version of the column bases that does not add over the subvariables, so results are less significant